config/config.json.sample

```diff
@@ -5,7 +5,26 @@
    "node_env": "local",
    "apidoc_url": "/apidoc",
    "apidoc_path":"/public/apidoc",
-
+   "file_path": {
```

Comment:
@sharmashbee , @shrawanTibil - Please provide values which can copy-paste from here to config.json